tock/hours/float.py

```diff
@@ -0,0 +1,147 @@
+import datetime as dt
+
+from .models import ReportingPeriod
+
+from employees.models import UserData
+from tock.utils import get_float_data, flatten
+
+# Variables.
+BASE_HOURS_PER_WEEK = 32.5
+BASE_HOURS_PER_DAY = 6.5
+BASE_DAYS_PER_WEEK = 5
+
+def float_tasks_for_view(request, kwargs):
```

Comment:
instead of passing in the `request` and `kwargs`, since you're only using one thing from each, just pass those in instead (`float_tasks_for_view(user_id, reporting_period)`) -- this way, it's more clear and easier to test (since you wouldn't have to mock a request object)